src/aegis_ai/osidb_bot/bot.py

```
@@ -363,9 +373,12 @@ async def do(self) -> None:
                     form_data=rh_cvss,
                 )
 
+            # successfully processed (we do not retry when only label creation fails)
+            self.retry_list.pop(self.cve, None)
+
         except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)